uint8 BOLT = 0
uint8 SMALL_GEAR = 1
uint8 LARGE_GEAR = 2
uint8 GEARBOX_TOP = 3
uint8 GEARBOX_BOTTOM = 4

uint8 object